config/api.yml

```diff
@@ -617,6 +617,35 @@
       :post:
       - :name: create
         :identifier: picture_new
+  :actions:
+      :description: Actions
+      :identifier: action
+      :options:
+      - :collection
+      :verbs: *gpd
+      :klass: MiqAction
+      :collection_actions:
+        :get:
+        - :name: read
+          :identifier: action_show
+        :post:
+        - :name: create
+          :identifier: action_new
```

Comment:
It should send the regular message from the generic.rb, its returning 
400, "\"message\": "Failed to add a new actions resource - Description can't be blank\""